presto-main/src/main/java/com/facebook/presto/execution/StageInfo.java

```diff
@@ -76,11 +77,16 @@ public URI getSelf()
     }
 
     @JsonProperty
-    public StageExecutionInfo getLatestAttemptExecutionInfo()
+    public Optional<StageExecutionInfo> getLatestAttemptExecutionInfo()
     {
         return latestAttemptExecutionInfo;
     }
 
+    public List<TaskInfo> getLatestTasks()
+    {
+        return latestAttemptExecutionInfo.isPresent() ? latestAttemptExecutionInfo.get().getTasks() : ImmutableList.of();
```

Comment:
nit: `return latestAttemptExecutionInfo.map(StageExecutionInfo::getTasks).orElse(ImmutableList.of());`